### PR TITLE
Import strings from android-l10n

### PR DESCRIPTION
**Pull request by Pike**
State: mozilla-l10n/android-l10n@413e05dba427ef8cbf9e5272ae5cc3b3ccdf91d3

In addition to the normal uplifts, this also adds Bulgarian, Hungarian, Tamil, Turkish.

CC @Delphine

### CODE REVIEW

File: app/src/main/res/values-es-rAR/strings.xml [hunk 1]
```diff
@@ -715,4 +730,66 @@
...
+    <string name="etp_panel_off">Las protecciones están OFF para este sitio</string>
+    <!-- Header for exceptions list for which sites enhanced tracking protection is always off -->
+    <string name="enhanced_tracking_protection_exceptions">La protección contra rastreo mejorada está deshabilitada para estos sitios</string>
+    <!-- About page Your rights link text -->
+    <string name="about_your_rights">Tus derechos</string>
+    <!-- About page link text to open open source licenses screen -->
+    <string name="about_open_source_licenses">Bibliotecas de código abierto que utilizamos</string>
+    <!-- Open source licenses page title
+    The first parameter is the app name -->
+    <string name="open_source_licenses_title">% s | Bibliotecas OSS</string>
```
Comment (ekager) [inline app/src/main/res/values-es-rAR/strings.xml]: Looks like there is a space here in the placeholder?

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 2]
```diff
@@ -730,4 +744,65 @@
...
+    <!-- Header for categories that are being blocked by current Enhanced Tracking Protection settings -->
+    <string name="enhanced_tracking_protection_blocking">封鎖</string>
+    <!-- Header for categories that are being not being blocked by current Enhanced Tracking Protection settings -->
+    <string name="enhanced_tracking_protection_not_blocking">不封鎖</string>
+    <!-- Category of trackers (social media trackers) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_social_media_trackers_title">社交媒體追蹤器</string>
+    <!-- Description of social media trackers that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_social_media_trackers_description">限制社群網站能在網路上追蹤您上網行為的能力。</string>
+    <!-- Category of trackers (cross-site tracking cookies) that can be blocked by Enhanced Tracking Protection -->
+    <string name="etp_cookies_title">跨網站追蹤 Cookie</string>
```
Comment (ekager) [inline app/src/main/res/values-zh-rTW/strings.xml]: Is this an incomplete translation>?

Comment (Delphine) [reply app/src/main/res/values-zh-rTW/strings.xml]: Nope, this is good to go :)

File: app/src/main/res/values-es-rAR/strings.xml (hunk 1 above)
Comment (Delphine) [reply app/src/main/res/values-es-rAR/strings.xml]: Updated in Pontoon, thanks!

**Pull request merged**